src/main/java/javapns/notification/management/SCEPPayload.java

```diff
@@ -25,7 +25,7 @@ public void setName(final String value) throws JSONException {
   public void setSubject(final String value) throws JSONException {
     final String[] parts = value.split("/");
     final List<String[]> list = new ArrayList<>();
-    for (final String part : parts) {
+    for (int i = 0; i < parts.length; i++) {
```

Comment:
@mlaccetti Because it more outlines the propose of the look but lets revert if you would like.